__tests__/ReactReduxUser.test.js

```diff
@@ -0,0 +1,33 @@
+import { expectSaga } from 'redux-saga-test-plan';
+
+import userSaga from '../store/sagas/user.sagas';
+import userReducer from '../store/reducers/user.reducer';
+import * as actions from '../store/actions/user.actions';
+
+//expectSaga.DEFAULT_TIMEOUT = 25000;
+//jest.setTimeout(15000);
+
+describe('User Saga', () => {
+  it('set UserData', () =>
+    expectSaga(userSaga)
+      .withReducer(userReducer)
+      .dispatch(
+        actions.login({
+          domain: 'dtappdemo.wpengine.com',
+          username: 'hansrasch',
+          password: 'Hrasch22...',
```

Comment:
@raschdiaz Is this your legit password?  If so, please change ASAP bc you'll be leaving the App Demo instance open for the Web